HISTORY.rst

```diff
@@ -1,6 +1,12 @@
 History
 =======
 
+0.12.0: 2017-01-19
```

Comment:
This file is maintained manually, and is usually updated by whoever ends up tagging a release for PyPI.